### PR TITLE
Add a example for call list option

### PR DESCRIPTION
**Pull request by tmirun**

### CODE REVIEW

File: source/api/index.md [hunk 1]
```diff
@@ -50,6 +50,12 @@ hexo.call('generate', {}).then(function(){
 });
 ```
 
+``` js
+hexo.call('list', { _: ['post'] }).then(function() {
+  console.log(arguments);
```
Comment (JLHwung) [inline source/api/index.md]: As a reference documentation, we should document what `arguments` are, i.e.
```js
const { exec } = require('child_process');
exec('cat *.js bad_file | wc -l', (error, stdout, stderr) => {
  if (error) {
    console.error(`exec error: ${error}`);
    return;
  }
  console.log(`stdout: ${stdout}`);
  console.log(`stderr: ${stderr}`);
});
```
is much better than
```js
const { exec } = require('child_process');
exec('cat *.js bad_file | wc -l', (...args) => {
  console.log(args);
})
```

Comment (tmirun) [reply source/api/index.md]: ups, i will remove console.log, hexo.call method is not returning any result

**Comment by tcrowe** [comment]
I wasn't really familiar with this api method so I went looking for examples. https://github.com/search?q=hexo.extend.console&type=Code

**Comment by tcrowe** [comment]
Thank you for your patience and your example @tmirun 👍🏼

**Pull request merged**